week1/armstrong.rb

```diff
@@ -9,5 +9,22 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
+    temp = n
+    arm=0
+    noDigits=0
```

Comment:
Prefer [snake_case](https://en.m.wikipedia.org/wiki/Snake_case) in Ruby and not camel case